apps/payments/next/app/[locale]/[offeringId]/[interval]/checkout/[cartId]/success/page.tsx

```diff
@@ -121,12 +120,10 @@ export default async function CheckoutSuccess({
             )}
           </div>
           <div className="flex items-center justify-between text-grey-400">
-            <PriceInterval
-              l10n={l10n}
-              amount={cart.latestInvoicePreview?.totalAmount}
-              currency={cart.latestInvoicePreview?.currency}
-              interval={cart.interval}
-            />
+            {l10n.getLocalizedCurrencyString(
+              cart.latestInvoicePreview?.totalAmount,
+              cart.latestInvoicePreview?.currency
+            )}
```

Comment:
This is right, but the underlying function still only uses locale `en` instead of the customers locale, so it still shows `$5.43` instead of `5,43 $` for German.

That said, I'm happy to fix this up in a separate PR, since PurchaseDetails also needs these changes. Could you file a follow up bug please?

Reply:
Filed - FXA-11378